### PR TITLE
CC-996: Ading a command to serviced to tell it when to start

### PR DESCRIPTION
**Pull request by kwalkerz**
Previously it started when --master or --agent was specified, but now the
existance of /etc/default/serviced often sets Master or Agent mode so we
were trying to start the server way too often...

Original complaint in JIRA ticket was that no command-line options did
not lead to help text being displayed, which used to happen.

### CODE REVIEW

File: cli/cmd/cmd.go [hunk 1]
```diff
@@ -136,6 +136,13 @@ func (c *ServicedCli) Run(args []string) {
 
 // cmdInit starts the server if no subcommands are called
 func (c *ServicedCli) cmdInit(ctx *cli.Context) error {
+	// DEBUG: KWW: Remove
```
Comment (wplu) [inline cli/cmd/cmd.go]: please remove all this debug

File: main.go [hunk 2]
```diff
@@ -44,7 +44,15 @@ func main() {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "WARNING: could not read default configs: %s\n", err)
 	}
-	cmd.New(api.New(), config).Run(os.Args)
+
+	// DEBUG: KWW: Restore original code
+	//cmd.New(api.New(), config).Run(os.Args)
+	mycmd := cmd.New(api.New(), config)
+	fmt.Println("--- os.Args ---")
```
Comment (wplu) [inline main.go]: please remove all this debug

**Comment by tgjonestx** [comment]
Retest this.

**Comment by wplu** [comment]
ship when test build passes

File: cli/api/api.go [hunk 3]
```diff
@@ -99,6 +99,25 @@ func GetOptionsRPCPort() string {
...
+func GetOptionsAgent() bool {
+	return options.Agent
+}
+
+// GetOptionsMasterPoolID returns the master pool ID from options
+func GetOptionsMasterPoolID() string {
+	return options.MasterPoolID
+}
+
+func GetOptionsMaxRPCClients() int {
```
Comment (wplu) [inline cli/api/api.go]: This is missing a comment - due to urgency of this fix, ignoring

**Pull request merged**